### ISSUE TITLE
Error when enabling extension

### ISSUE THREAD

**Issue by willnode** [FriendsOfFlarum/best-answer#51]
I notice this repo has recent pushes so I'd like to report.

I have a problem when enabling the extension. using "version": "1.1.2"

```
TypeError: explode(): Argument #2 ($string) must be of type string, array given in D:\GIT\forum\vendor\fof\best-answer\migrations\2021_08_15_migrate_reminder_settings.php:25
Stack trace:
#0 D:\GIT\forum\vendor\fof\best-answer\migrations\2021_08_15_migrate_reminder_settings.php(25): explode(',', Array)
#1 [internal function]: Illuminate\Filesystem\Filesystem::{closure}(Object(Illuminate\Database\Schema\MySqlBuilder))
#2 D:\GIT\forum\vendor\flarum\core\src\Database\Migrator.php(202): call_user_func(Object(Closure), Object(Illuminate\Database\Schema\MySqlBuilder))
#3 D:\GIT\forum\vendor\flarum\core\src\Database\Migrator.php(133): Flarum\Database\Migrator->runClosureMigration(Array)
```

https://github.com/FriendsOfFlarum/best-answer/blob/732a45f75195ccb4e67c629bbf29dd43958e5f90/migrations/2021_08_15_migrate_reminder_settings.php#L23-L25

What's the last stable version btw? (nvm downgrading to 1.0.1 works to me).

**Comment by n0099** [comment]
workaround:
```sql
INSERT INTO `prefix_settings` (`key`, `value`) VALUES ('fof-best-answer.remind_tag_ids', NULL);
```

**Comment by imorland** [comment]
Fixed from `1.1.3` onwards
